### DISCUSSION TITLE
Feature Request: "Follow Projects"

### DISCUSSION THREAD

**Discussion by thinkenvy** [community/community#65629] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

It would be great if there was a way to "follow" projects, so I get notified when something is changed.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by GhostOf0days** [comment]
Do you mean Github projects themselves or repos? If it's a project you could click the star and the bell (notifications).

**Comment by thinkenvy** [comment]
I mean, for instance, I would like to be notified when something changes on this project: https://github.com/orgs/webwhiz-ai/projects/3

I don't see any star or bell icon to click for notifications.
